Jenkinsfile

```diff
@@ -19,7 +18,7 @@ pipeline {
     environment {
         RUNNING_ON_CI = 'true'
         VERSION_NUMBER = "${params.VERSION_NUMBER}"
-        BUILD_NUMBER = "${params.BUILD_NUMBER}"
+        BUILD_NUMBER = sh(script: "echo `date +%y%m.%d.%H%M`", returnStdout: true).trim()
```

Comment:
why are we changing versions to dates?

this will lead to non-deterministic builds.

Reply:
> why are we changing versions to dates?

No, just the `build number` is changing to timestamp derivative. The version stays the same.
```
VERSION_NUMBER = "${params.VERSION_NUMBER}"
```
Below is how it looks like. More convenient, it gives the date when it's built and unique. TestFlight complains if you try to upload the same build number which is already uploaded.

![image](https://user-images.githubusercontent.com/6256269/139852814-237e31d2-3542-48ba-b3a7-deb9ccf8c8bb.png)